fvtest/porttest/omrvmemTest.cpp

```diff
@@ -388,6 +388,122 @@ TEST(PortVmemTest, vmem_test1)
 	reportTestExit(OMRPORTLIB, testName);
 }
 
+/**
+ * Get all the page sizes and make sure we can allocate a memory chunk for each page size.
+ * This particular test is testing the flag OMRPORT_VMEM_MEMORY_MODE_FILE_HANDLE where it
+ * forces default_pageSize_reserve_memory to mmap using a file handle.
+ * 
+ * Checks that each allocation manipulates the memory categories appropriately.
+ */
+TEST(PortVmemTest, vmem_test_file_handle)
+{
+	portTestEnv->changeIndent(1);
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	const char *testName = "omrvmem_test_file_handle";
+	char *memPtr = NULL;
+	uintptr_t *pageSizes = NULL;
+#if defined(J9ZOS390)
+	uintptr_t *pageFlags = NULL;
+#endif /* J9ZOS390 */
+	int i = 0;
+	struct J9PortVmemIdentifier vmemID;
+	char allocName[allocNameSize];
+	int32_t rc = 0;
+	char *lastErrorMessage = NULL;
+	int32_t lastErrorNumber = 0;
+
+	reportTestEntry(OMRPORTLIB, testName);
+
+	/* First get all the supported page sizes */
+	pageSizes = omrvmem_supported_page_sizes();
+#if defined(J9ZOS390)
+	pageFlags =
+#endif /* J9ZOS390 */
+		omrvmem_supported_page_flags();
```

Comment:
shouldn't this call be inside the `#if defined`?  It looks like you copied most of this from another test but I would still clean up that in this new version.